sources/meso.json

```diff
@@ -0,0 +1,10 @@
+{
+	"id": 1108756907,
+	"fullname": "Mesoshapes",
+	"name": "meso",
+	"prefix": "meso",
+	"key":"",
+	"url": "https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md",
```

Comment:
Added information about mesoshapes in https://github.com/whosonfirst-data/whosonfirst-data/pull/554.